CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.5...4.0)
```

Comment:
Without a range of CMake versions, the version given is the _exact_ version of the CMake policy that is used regardless of the version of CMake installed. Newer versions of modern CMake have dropped support for very old releases (like `v3.0`) and so Delphes without this patch can not build with them.